### PR TITLE
Update to last Emitter update

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Needed for [this PR](https://github.com/rust-lang/rust/pull/44636#issuecomment-336191781).

However, this'll break until the mentionned PR gets merged, and the mentionned PR requires this one to be merged otherwise tests won't pass. So much fun. :)

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -510,6 +510,7 @@ pub fn format_input<T: Write>(
     let silent_emitter = Box::new(EmitterWriter::new(
         Box::new(Vec::new()),
         Some(codemap.clone()),
+	false,
```
Comment (topecongiro) [inline src/lib.rs]: Could you please fix the indentation here?

Comment (GuillaumeGomez) [reply src/lib.rs]: Oups, my bad.

**Comment by topecongiro** [comment]
Created [emitter](https://github.com/rust-lang-nursery/rustfmt/tree/emitter) branch.

**Comment by GuillaumeGomez** [comment]
Thanks! (and updated)

**Comment by GuillaumeGomez** [comment]
The corresponding rust PR has been merged so tests should be working back again in here. Once this PR merged, I'll re-enable rustfmt in the compiler.

**Comment by nrc** [comment]
There hasn't been a new nightly for a few days. I guess we have to wait for that to get fixed before doing anything here.

**Pull request merged**